ci/azure-piplines.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     displayName: Environment Info
 
   - bash: |
-      pytest -n auto --pyargs hera_cal -W "ignore" -W "default:::pyuvdata"
+      pytest -n auto --pyargs hera_cal
```

Comment:
Why are we reverting to not ignoring warnings here? Because hera_cal now has far less warnings?

Reply:
Yes, and because I was finding that it filtered pyuvdata warnings, despite earlier testing that suggested it did not.